include/hse/hse.h

```diff
@@ -514,12 +504,11 @@ hse_kvdb_kvs_drop(struct hse_kvdb *kvdb, const char *kvs_name);
  * @returns Error status.
  */
 hse_err_t
-hse_kvdb_kvs_open(
-    struct hse_kvdb *  kvdb,
-    const char *       kvs_name,
-    const size_t       paramc,
-    const char *const *paramv,
-    struct hse_kvs **  kvs_out);
+hse_kvdb_kvs_open(struct hse_kvdb *kvdb,
```

Comment:
Not a huge fan of the first parameter like that

Reply:
Fixed.